javascript/net/grpc/web/grpcwebstreamparser.js

```diff
@@ -175,16 +175,28 @@ Parser.prototype.error_ = function(inputBytes, pos, errorMsg) {
 
 
 /**
+ * Parse the new input.
+ *
+ * Note that there is no Parser state to indicate the end of a stream.
+ *
+ * @param {string|!ArrayBuffer|!Uint8Array|!Array<number>} input The input data
  * @throws {!Error} Throws an error message if the input is invalid.
- * @override
```

Comment:
Sorry another issue: apparently you need `@override` here because now the spec differs

Reply:
Fixed